nerfstudio/data/datamanagers/full_images_datamanager.py

```diff
@@ -132,27 +132,29 @@ def cache_images(self, cache_images_option):
                 continue
             distortion_params = camera.distortion_params.numpy()
             image = data["image"].numpy()
+            distortion_params = np.array(
+                [
+                    distortion_params[0],
+                    distortion_params[1],
+                    distortion_params[4],
+                    distortion_params[5],
+                    distortion_params[2],
+                    distortion_params[3],
+                    0,
+                    0,
+                ]
+            )
 
             if camera.camera_type.item() == CameraType.PERSPECTIVE.value:
-                distortion_params = np.array(
```

Comment:
I think we should keep this distortion_params update inside PERSPECTIVE type if-clause, otherwise, it may affect the FISHEYE to read the correct order of parameters.

Reply:
+1

Reply:
Good call. Thanks